src/engine/racer.rs

```diff
@@ -153,9 +142,6 @@ pub fn collapse_whitespace(text: &str) -> String {
   Regex::new(r"\s+").unwrap().replace_all(text, " ")
 }
 
-unsafe impl<'a> Sync for Racer<'a> {}
-unsafe impl<'a> Send for Racer<'a> {}
```

Comment:
It does need it, I moved it up and wrote a comment on why it is safe, since I expected to be able to remove it :(

Sorry for the confusion though, I should've left it there.